cfme/fixtures/v2v.py

```diff
@@ -11,12 +12,31 @@
 from cfme.utils.log import logger
 
 
+FormDataVmObj = namedtuple(
+    'FormDataVmObj', ['form_data', 'vm_obj']
+)
+
+
 @pytest.fixture(scope='function')
 def v2v_providers(request, second_provider, provider):
     """ Fixture to setup providers """
+    V2vProviders = namedtuple(
+        'V2vProviders', ['vmware_provider', 'rhv_provider']
+    )
+    vmware_provider, rhv_provider = None, None
+    for provider in [second_provider, provider]:
+        if provider.one_of(VMwareProvider):
+            vmware_provider = provider
+        elif provider.one_of(RHEVMProvider):
+            rhv_provider = provider
+        else:
+            pytest.skip("Provider {} is not a valid provider for v2v tests".
+                format(provider.name))
+    if not vmware_provider or not rhv_provider:
+        pytest.skip("Something went wrong with VMware or RHV provider, skipping test.")
     setup_or_skip(request, second_provider)
     setup_or_skip(request, provider)
```

Comment:
We should be using the values of the namedtuple here instead of `provider` and `second_provider`